pivotal-import/pivotal_import.py

```diff
@@ -325,7 +347,13 @@ def build_entity(ctx, d):
         d["group_ids"] = [group_id] if group_id is not None else []
 
     entity = {k: d[k] for k in select_keys[type] if k in d}
-    return {"type": type, "entity": entity, "parsed_row": d}
+    return {
+        "type": type,
+        "entity": entity,
+        "iteration": iteration,
```

Comment:
You could probably avoid changing all those tests by conditionally adding these keys to the returned map if they are not `None`

Reply:
You've caught me on the path of my career arc where "same shape all the time is better than optional parts of a shape."

Since you've done an approval + comment I'm going to leave this be, but if it frustrates you during future work, feel free to go the other route.